Tests/search_and_install_packs.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import print_function
+
+import os
+import ast
+import demisto_client
+from threading import Thread, Lock
+from demisto_sdk.commands.common.tools import print_error, print_color, LOG_COLORS, is_file_path_in_pack, \
+    get_pack_name, run_threads_list
+from Tests.test_content import ParallelPrintsManager
+
+
+def get_pack_id_by_path(path):
+    if is_file_path_in_pack(path):
+        return get_pack_name(path)
+    return ''
+
+
+def get_pack_from_results(search_results, pack_id):
+    for pack in search_results:
+        if pack.get('id') == pack_id:
+            return pack
+    return {}
+
+
+def search_pack(client, prints_manager, thread_index, pack_id):
+    """ Make a pack search request.
+
+    Args:
+        client (demisto_client): The configured client to use
+        prints_manager (ParallelPrintsManager): Print manager object
+        thread_index (int): The thread index
+        pack_id (string): The pack ID
+
+    Returns:
+        (dict): Returns the pack metadata if found, or empty dict otherwise.
+    """
+
+    host = client.api_client.configuration.host
+    prints_manager.add_print_job('\nMaking "POST" request to server - "{}" to search pack {}.'.format(host, pack_id),
+                                 print, thread_index)
+
+    # make the search request
+    response_data, status_code, _ = demisto_client.generic_request_func(self=client,
+                                                                        path='/contentpacks/marketplace/search',
+                                                                        method='POST',
+                                                                        body={"packsQuery": pack_id, "size": 3},
```

Comment:
in order to test the search, we need to search the pack with its display name, not the id. thats because the user will see the display name and is expected to search that string, rather than the id.

BTW whats the meaning of the size here? why it is 3?